ratpack.gradle

```diff
@@ -123,10 +123,10 @@ ext {
     "ratpack-core", "ratpack-groovy", "ratpack-guice", "ratpack-session", "ratpack-test", "ratpack-groovy-test",
     "ratpack-manual", "ratpack-gradle", "ratpack-handlebars", "ratpack-remote", "ratpack-remote-test", "ratpack-jackson",
     "ratpack-codahale-metrics", "ratpack-h2", "ratpack-thymeleaf", "ratpack-rx", "ratpack-hikari", "ratpack-newrelic",
-    "ratpack-pac4j", "ratpack-hystrix", "ratpack-spring-boot", "ratpack-config"
+    "ratpack-pac4j", "ratpack-hystrix", "ratpack-spring-boot", "ratpack-config", "ratpack-benchmark"
```

Comment:
I don't think we'll want to publish anything from this subproject, so this can be removed. Which means it can be removed below as well.